lib/logitech_receiver/descriptors.py

```diff
@@ -213,7 +213,14 @@ def _D(name, codename=None, kind=None, wpid=None, protocol=None, registers=None,
 # Mice
 
 _D('Wireless Mouse M175', protocol=2.0, wpid='4008')
-_D('Wireless Mouse M185')
+_D('Wireless Mouse M185 new', codename='M185n', protocol=4.5, wpid='4054',
+				settings=[
+							_FS.lowres_smooth_scroll(),
+				])
+_D('Wireless Mouse M185 old', codename='M185o', protocol=4.5, wpid='4055',
+				settings=[
+							_FS.lowres_smooth_scroll(),
+				])
```

Comment:
Yes, Logitech does indeed re-utilize model numbers a lot :(

Anyway, I suspect that feature auto-detection has its issues too. We have to look carefully to this, meanwhile, let´s add descriptors for everyone. I PR #342  and marked it as dependent on this one since it needs USB nano receiver recognition. Thanks for your effort. 

Now we wait for @Lekensteyn to have some time to merge.